src/staff_augmentation/components/GetFreeQuote/FreeQuote/Textarea/index.js

```diff
@@ -0,0 +1,33 @@
+import { useCallback, useState } from 'react'
+
+import classes from './styles.module.scss'
+
+function Textarea ({ onChange, maxLength, ...props }) {
```

Comment:
Don't copy/paste this component. You can move it to `src/main_app/components/TextArea`, so that you can share it in both components that use it.